build.sh

```diff
@@ -23,7 +23,7 @@ sudo apt-get install -y \
 
 sudo mkdir /usr/local/php7
 
-git clone https://github.com/php/php-src.git
+git clone -b PHP-7.0.0 --depth=1 https://github.com/php/php-src.git
 cd php-src
 git checkout PHP-7.0.0
```

Comment:
I see, but the version is hard coded in the script, so it looks natural to me to only get the version (maybe even downloading the .zip would be faster)..

`git clone --depth=1` took 23s in my case, without `--depth=1` it is 5mins .. so for some time, deleting/re-fetching is a good decision ;)